settings.py

```diff
@@ -51,3 +51,11 @@
     GRADER_ROOT = ENV_TOKENS.get('GRADER_ROOT')
 
     SANDBOX_PYTHON = ENV_TOKENS.get('SANDBOX_PYTHON', '/opt/edx/bin/sandbox-python')
+
+
+if ENV_TOKENS.get(SENTRY_DNS):
+    import raven
```

Comment:
Could you please move the import statement to the beginning of the file? Thanks

Reply:
Done